profiles/events.py

```diff
@@ -16,16 +16,23 @@ def send_update_events(publisher: EventPublisher) -> Callable[..., None]:
     def event_handler(sender: Flask, changes: tuple) -> None:  # pylint:disable=unused-argument
```

Comment:
`sender` type needs fixing.

Reply:
Ah, was corrected in https://github.com/elifesciences/profiles/pull/53 but not here. Now done.

Reply:
Was referring to `sender: Flask`, which is now `sender: Any`.